hist/hist/src/TH1.cxx

```diff
@@ -6202,6 +6202,36 @@ Bool_t TH1::Multiply(const TH1 *h1, const TH1 *h2, Double_t c1, Double_t c2, Opt
    return kTRUE;
 }
 
+////////////////////////////////////////////////////////////////////////////////
+/// @brief Normalize a histogram to its integral or to its maximum.
+/// @note Works for TH1, TH2, TH3, ...
+/// @param option: normalization strategy ("", "max" or "sum")
+/// - "": Scale to `1/(sum*bin_width)`.
+/// - max: Scale to `1/GetMaximum()`
+/// - sum: Scale to `1/sum`.
+/// In case the norm is zero, it raises an error.
```

Comment:
We need a blank line to separate the explanation of `@param` from the rest of the text.
```suggestion
/// - sum: Scale to `1/sum`.
///
/// In case the norm is zero, it raises an error.
```